noir-projects/aztec-nr/aztec/src/state_vars/private_set.nr

```diff
@@ -40,25 +41,12 @@ impl<Note, Context> PrivateSet<Note, Context> {
     // docs:end:new
 }
 
-impl<Note, let N: u32> PrivateSet<Note, &mut PublicContext>
-where
-    Note: NoteInterface<N> + NullifiableNote,
-{
-    // TODO: This function is still around because of a stale blacklist token. It should most likely be nuked. If you
-    // need this functionality use partial notes instead.
-    // docs:start:insert_from_public
-    pub fn insert_from_public(self, note: &mut Note) {
-        create_note_hash_from_public(self.context, self.storage_slot, note);
```

Comment:
I've nuked this and instead I just push the note hash into the context in the blacklist token directly. This made sense to do here because computing the note hash became more straightfroward:
<img width="1110" alt="image" src="https://github.com/user-attachments/assets/916b0f4b-a57a-4716-afb5-6e46b80efcf8" />